test/expected/011_parallel_test_1.out

```diff
@@ -0,0 +1,377 @@
+SET max_parallel_workers_per_gather = 8;
+ERROR:  unrecognized configuration parameter "max_parallel_workers_per_gather"
```

Comment:
So, this test expectation is for versions when this setting did not exist, so PG 9.5 and before, and we have another change in plans after PG 11.
On another extension, I used something like that to make it explicit which version goes with which expectation:
```suggestion
-- different results are expected based on PostgreSQL version
SELECT current_setting('server_version_num')::int < '90600'::int AS before_v9_6, current_setting('server_version_num')::int < '110000'::int AS before_v11;
 before_v9_6 | before_v11
-------------+------------
 t           | t
(1 row)
SET max_parallel_workers_per_gather = 8;
ERROR:  unrecognized configuration parameter "max_parallel_workers_per_gather"
```

Do you think that would be something useful to add?

Reply:
I'm not sure how it can help, but I added it :)

Reply:
Actually I just removed all expected outputs of Postgres before 11 as they are not supported anymore. I also removed old versions from Github actions and added Postgres 15.

Reply:
Indeed, better to remove them, thanks :)

> I'm not sure how it can help, but I added it :)

Well, they don't really handle a likely test case, I used this for two reasons:
- it made it more convenient for me to identify later which expected file goes with which PostgreSQL version
- it makes the test fail in the (unlikely) case that a future PostgreSQL version ends up with a "regression" to an output we'd expect for a previous version only (like maybe if the `max_parallel_workers_per_gather` was removed in a later version for example)